docker/entrypoint.sh

```diff
@@ -4,8 +4,8 @@ cat <<EOF > /etc/orchestrator.conf.json
 {
   "Debug": true,
   "ListenAddress": ":3000",
-  "MySQLTopologyUser": "${ORC_TOPOLOGY_USER:-:orchestrator}",
-  "MySQLTopologyPassword": "${ORC_TOPOLOGY_PASSWORD:-:orchestrator}",
```

Comment:
That's actaully shell code, which generates a sample config if there is no config file already present at `/etc/orchestrator.conf.json`. The bug was there since my initial commit, I just found it now.